povcalnet.ado

```diff
@@ -258,8 +259,26 @@ qui {
 		}
 	}
 	
-	*---------- Poverty line
-	if ("`povline'" == "") local povline = 1.9
+	*---------- Poverty line/population share
+	if ("`popshare'" != ""){
+		if ("`povline'" != ""){
+			oi disp as err "povline and popshare cannot be used at the same time"

```

Comment:
I Think this should be `noi` rather than `oi`

Reply:
Awful typo, I'll fix it right away